src/App.js

```diff
@@ -1,79 +1,19 @@
 import "./App.css";
-import {useEffect, useState} from "react";
-import NavBar from "./NavBar";
-import ClassCard from "./ClassCard";
-import Footer from "./Footer";
-
-const NAV_LINKS = [
-  {title: "Classes", url: "/classes"},
-  {title: "Chats", url: "/chats"},
-  {title: "Profile", url: "/profile"}
-];
-
-const CLASSES = [
-  {
-    id: 0,
-    title: "COMP_SCI 211-0",
-    description: "Fundamentals of Computer Programming II",
-    members: 67,
-    url: "/classes/comp-sci-211-0"
-  },
-  {
-    id: 1,
-    title: "COMP_ENG 203-0",
-    description: "Introduction to Computer Engineering",
-    members: 48,
-    url: "/classes/comp-eng-203-1"
-  },
-  {
-    id: 2,
-    title: "MATH 228-1",
-    description: "Multivariable Differential Calculus for Engineering",
-    members: 61,
-    url: "/classes/math-228-1"
-  },
-  {
-    id: 3,
-    title: "GEN_ENG 206-1",
-    description: "Honors Engineering Analysis 1",
-    members: 56,
-    url: "/classes/gen-eng-206-1"
-  },
-  {
-    id: 4,
-    title: "CONDUCT 393-0",
-    description: "Orchestral Organizations",
-    members: 77,
-    url: "/classes/conduct-393-0"
-  }
-];
+import {BrowserRouter, Route, Routes} from "react-router-dom";
+import PageLayout from "./navigation/PageLayout";
+import Classes from "./classes/Classes";
+import Class from "./classes/class/Class";
```

Comment:
since `Class` is an entirely different page from `Classes`, I would recommend putting them on the same directory level instead of having `Class` be a subdirectory of `Classes`.
my recommendation is to have a src/pages/ directory, with src/pages/classes and src/pages/class subdirectories.
src/pages/classes would contain `Classes.js` and other related files
src/pages/class would contain `Class.js` and other related files